src/routes/log.js

```diff
@@ -11,6 +11,13 @@ function log(req, res) {
   });
 }
 
-router.get('/', log);
+async function write(req, res) {
```

Comment:
No, Raptor's only option of consuming data is by parsing logcat. Since there is no memory information in logcat, we write the data to logcat and parse it back out by writing entries:

```
I/PerformanceMemory( 1293): verticalhome.gaiamobile.org|uss|20.277 
I/PerformanceMemory( 1301): system.gaiamobile.org|uss|61.613 
I/PerformanceMemory( 1307): system.gaiamobile.org|pss|67.629 
I/PerformanceMemory( 1308): system.gaiamobile.org|rss|80.125 
I/PerformanceMemory( 1306): verticalhome.gaiamobile.org|pss|25.856 
I/PerformanceMemory( 1300): verticalhome.gaiamobile.org|rss|38.027
```